### DISCUSSION TITLE
place to discuss and ask for help

### DISCUSSION THREAD

**Discussion by pm100** [emilk/egui#4795] [General]
egui has several places where people ask for help.
- here
- github issues
- reddit r/rust
- stackoverflow

This spreads the egui comminuty a bit thin, and leads to frustration too. egui is not simple and the docs are okish but once you start trying to build real things you end up needing help. 

It would be good if there was one preffered place to post questions when you are stuck. Then put it  prominently in the doc and readme.

maybe r/egui, discord, or here in discussions

stackoverflow is not good becuase you end up arguing with its automatied mod system about whats an acceptable question

**Comment by YgorSouza** [comment]
The main places to ask questions are this discussion tab and the Discord server, as mentioned in the Readme: <https://github.com/emilk/egui?tab=readme-ov-file#quick-start>.

**Comment by pm100** [comment]
OK, thanks. I missed the discord server. Which is the preferred one (I will redirect people asking in reddit and stackoverflow)

**Reply by emilk** [reply]
This GitHub Discussion forum is preferable since answers can be found later by searching for them.

**Comment by pm100** [comment]
Thx
